publish

```diff
@@ -24,7 +24,7 @@ function nodeInstall {
 }
 
 function nodeCleanup {
-  printf "Removing node_modules and etc for $1 lambda"
+  printf "Removing node_modules and etc for $1 lambda\n"
```

Comment:
I prefer to use `printf` over `echo` in bash scripts, I have found it to be more intuitive and consistent. http://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo#65819 explains their differences pretty in depth.